src/main/java/app/backend/service/VideoService.java

```diff
@@ -28,11 +28,12 @@ public VideoService(GridFsTemplate gridFsTemplate, GridFsOperations gridFsOperat
         this.gridFsOperations = gridFsOperations;
     }
 
-    public String store(MultipartFile file, String crossroadId, String timeIntervalId) {
+    public String store(MultipartFile file, String crossroadId, String startTimeId, Integer duration) {
         DBObject metadata = new BasicDBObject();
         metadata.put("crossroadId", crossroadId);
-        metadata.put("timeIntervalId", timeIntervalId);
+        metadata.put("startTimeId", startTimeId);
         metadata.put("type", file.getContentType());
+        metadata.put("duration", duration);
```

Comment:
Dodaj do schematu DB (przy okazji możesz userów poprawić - imię i nazwisko w schemacie)

Reply:
To jest dodane w Video, tylko jako time